tests/service-account-test.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "service_account",
+  "project_id":"project_id",
+  "private_key_id":"test_kid",
+  "private_key":"-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDG7fLXGssDfkpU\nK3v0oEggl3mVw5rVVl9nXvI38r22WDJCdlSi1Kd3KrvxaN+Mt9FOgzDqoKoYH2b9\nNYf60J9TE0YGcX6G9XrRRii2oyaO2sYiJAjzlPwZo3vR54+hmeg0GjyObBOITGx+\nZDjJPchkDs36tcjX8JOGBrBaPY+oPrLr7hFFzgp0SHmM4/lZ8MJzfpcEtpHvKzy8\npHUJlX8NCR86GYqN5cLDu7HsvcqtqmIML76auMgcWPea5Wjk98JieEEXN5lK0DW6\nsN2Oi3MLbZu7rKZJ1gLylC6uypa9LRJOVFgj82fOse7Bp0NBgGnjr3XN1QUOAGCS\nsInSKYU/AgMBAAECggEAWyMes2/h/Jq6YO9/HablBJGBMZzo7b5hfRFhtUIvqj+j\n3xEpW2RDyPO6ITKj4GtCqE6wdX3gD6crXuxMfRthMwVMep06k4gZmZEkC/CZNK+E\nQJXzx+zExtZAXv1Qr3+8g0pV3gYjuLkSp6Ew5vm4OicSNT9dYZklbSzZVK6Jh1FL\nhsPk8Jp32xcBUcFnFqr45ay9yfsOvfViUhdNWXFgJKL0G3OCChFY2kce+fbj7lDk\nOh5FWWa7dCYAmYNZy0HNbVx/ww/CFFamiYphBU0Gg+0uvTyoLEuF1T3ssM3E5vzb\n3jUCjnRrnsTdUnqWWDTKYJWaWLYR/bngeTsEpkSYiQKBgQDllOBDD9eNNIg3yFbj\nBmGbuxEkO/WFx3/hyAd1LkhdmApJY/reVymOfeOBosCGVDC3Qt8wflUE6Y6Nzn0y\nKCEbrfc9gTOL69O0LfnFy0WikNMCzqUJAFEAQRhyMp25hvAtfrxTbIeN0WNLjwDS\n9jHoYJHOS6aumzOV2lB8GYQdxQKBgQDd0hwBEvNfjlJGCp0O5BQHF1PZQ2e6pOJs\nf5pAD7KC1cARQJKztBWgwVaufCimOpYUlpiu2rqARL424ktZlEP+hhrX2aZj8g68\ndtrLvuPMZTBHNPqtNu9baSPvChT1ha/yrIslVXjhA3NRCUO9k+F53pXbkSZGQBZL\ndyDZhXGrMwKBgQDTgzy2LDM/0cUp93YtROTKkCczxdXnAa0M+7f7Or/LEtdvVCB9\nlbogoFTpS2OqnogfXwm3aLy1gOQoO7RWcGhIUxd038L4xzVNTApRM3ydUGZHsNCV\noWf/HvoBxCZSFkLS929UQKoGe4HKzB5LPi0u1UGf2UzgkvaMugzquKtirQKBgAla\nEVouPVp6+pb/XY3CKeH+psdTIy1oRC+E1bD7uG/xqQ5RJZ4z6nGDlS74BxKzB57I\nlplQmWpM+6P+uGSHbrJtXvZYs74k92AUtra+ToZQgc+OuT7SQRcegKhUSXvsbfWS\nNq+VQepipdr1xAhbErd7nd/K580waIA3/oeNK7SnAoGASTffZsSi+XyogHOTrLCT\nFHL8i9O3LF/I9wZsGgdqMq+XtBYG1cMhn4g4GBNB7b4GgIEhe5Fzw4opnA9jyNzz\nRITrkpuvmJathvim4pPophEfcARtlu9RJzELZDV8SRbXRgVtLT2XS16bDyFS1n+R\nqSTMvJOdu+3RWf7LGKNIrNE=\n-----END PRIVATE KEY-----",
```

Comment:
Question - do you feel like we should expose the test credentials for integration in a Github secret? I think the likelihood of misuse is low, but still something worth considering?

Reply:
The service-account-test.json file does not contain real keys, it is just for the new unit test of the Credentials ::new and ::from_file operations.